knot-thing/src/main/java/com/cesar/knot_sdk/KNoTMessageParser.kt

```diff
@@ -92,4 +94,12 @@ class KNoTMessageParser {
 
         return KNoTMessageRequestData(id, sensorIds)
     }
+
+    fun parseSchemaStatus(knotSchema : String) : KNoTMessageSchemaResp {
+        val receivedJson : JsonElement = JsonParser.parseString(knotSchema)
+        val id = receivedJson.asJsonObject.get(ID).asString
+        val error = receivedJson.asJsonObject.get(ERROR).toString()
```

Comment:
I believe you could change the first line to end with asJsonObject instead of calling it twice down bellow. Look at your previous commit.

Reply:
Done.